test/README.md

```diff
@@ -0,0 +1,61 @@
+# Test File Directory
+
+This directory contains all test files for this project.
+
+## Running Tests
+
+Tests are automatically run via CI/CD setup on GitHub actions, and can always be invoked via `make test` from the root of
+this repository.
+
+To target specifically Vim or Neovim, the `test-vim` and `test-nvim` target is called instead.
+
+You'll need Vim and/or Neovim installed. The Makefile will automatically detect whether they're installed, and run the tests
+in the respective environment.
+
+## Adding New Tests
+
+This project uses a framework to test its Vim Script: [vader.vim](https://github.com/junegunn/vader.vim). Before adding new
+tests, it's definitely recommended to read its documentation.
+
+When adding new tests, only one specific feature should be tested within one test file. This is to avoid hard-to-read,
+hard-to-debug test files as well pipeline output. It furthermore makes it simpler to expand already existing tests.
+
+Afterwards, `Include` your newly added test file in `test.vader` and make sure all tests are passing.
```

Comment:
One thing that popped up regarding `Include`:

With the old structure, each individual file was one test suite. Now, that we *plan to* include files, that collapses down to one big test suite.

Let me know what you think -- personally, I prefer the more detailed logging that we currently have, as it makes debugging easier. However, I can also see why one would prefer one large suite.

Screenshots of the two logs attached below.

![Logs with `Include` directive](https://user-images.githubusercontent.com/71897876/192764933-4430e724-a691-4f88-8c03-a09a6e9e3529.png)

![Individual suites (w/ new directory structure)](https://user-images.githubusercontent.com/71897876/192765081-eeae1f5c-857f-4f8c-85af-7cde3f299dfa.png)

Reply:
I think I agree, actually — having multiple suites is nicer in terms of granularity.

Reply:
Will push the appropriate changes 👍